app/controllers/api/v1/line_items_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Api::V1
+  class LineItemsController < ApplicationController
+    before_action :set_line_item, only: %i[show update]
+
+    def index
+      @line_items = LineItem.all
+      render json: @line_items
+    end
+
+    def show
+      render json: @line_item
+    end
+
+    def update
+      @line_item.update(line_item_params)
+      @line_item.save!
+    end
+
+    def destroy
+      @line_item = LineItem.destroy(params[:id])
+      render json: @line_item
+    end
+
+    def create
+      @line_item = LineItem.new(line_item_params)
+      @line_item.save!
+      render json: @line_item
```

Comment:
I see here the new controller action and the create controller action is mixed.  That could be a json api thing, should discuss with Jeff.